troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+           Universal time: Tue 2020-03-10 18:20:02 UTC
+                 RTC time: Tue 2020-03-10 18:36:53
+                Time zone: UTC (UTC, +0000)
+System clock synchronized: no
+              NTP service: active
+          RTC in local TZ: no
+```
+
+This can be caused due to issue on Time sync. This can be fixed by
+ 
+1. Add  chrony.conf file and encode it as base64. For example: 
+
+```
+cat << EOF | base 64
+server registry.kni7.cloud.lab.eng.bos.redhat.com
+driftfile /var/lib/chrony/drift
+logdir /var/log/chrony
+log measurements statistics tracking
+keyfile /etc/chrony.keys
+commandkey 24
+local stratum 10
+initstepslew 20 registry.kni7.cloud.lab.eng.bos.redhat.com
+logchange 0.5
+allow fd35:919d:4042:2::1000 #IP-address-of-NTP-server
+EOF 
+[text-in-base-64] 
+```
+2. Create the MachineConfig file, replacing the base64 string with the one you just created yourself. This example adds the file to master nodes. You can change it to worker or make an additional MachineConfig for the worker role: 
+```
+cat << EOF > ./99_masters-chrony-configuration.yaml 
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  creationTimestamp: null
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-etc-chrony-conf
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,[text-in-base-64] 
+          verification: {}
+        filesystem: root
+        group:
+          name: root
+        mode: 420
+        path: /etc/chrony.conf
+        user:
+          name: root
+    systemd: {}
+  osImageURL: ""
+```
+3. Make a backup copy of the configuration file. 
+
+4. If the cluster is not up yet, generate manifest files, add this file to the openshift directory, then continue to create the cluster.
+
+5. If the cluster is already running, apply the file as follows: 
+
+```
+# oc apply -f ./masters-chrony-configuration.yaml
+sudo timedatectl
+               Local time: Tue 2020-03-10 19:10:02 UTC
+           Universal time: Tue 2020-03-10 19:10:02 UTC
+                 RTC time: Tue 2020-03-10 19:36:53
+                Time zone: UTC (UTC, +0000)
+System clock synchronized: yes
+              NTP service: active
+          RTC in local TZ: no
+```
+
+
+# IPv6 Disconnected Install Troubleshooting
+
+In this document we will analyze common scenarios of errors/failures which can cause the cluster installation to fail. 
+
+## Common failures Basic checks before Installing
+
+Below are some precautions we should follow before even attempting the OCP4 Ipv6 deployments. 
+
+* [Ipv4-BM-IPI](https://github.com/openshift-kni/baremetal-deploy/blob/master/install-steps.md#preparing-the-provision-node-for-openshift-install) has some pre-requisites which will be applied fpr ipv6 deployments as well. 
+
```

Comment:
What checks should I do? Should I bother with checks? I assume they followed them and got a specific error. I wouldn't both with this here.

Reply:
Im removing this section as per above suggestion